apps/designer/app/routes/designer/$id.tsx

```diff
@@ -1,9 +1,9 @@
 import { useActionData, useLoaderData } from "@remix-run/react";
 import { ActionFunction, LoaderFunction } from "@remix-run/node";
-import type { Project, Asset } from "@webstudio-is/prisma-client";
 import { toast } from "@webstudio-is/design-system";
 import { Designer, links } from "~/designer";
 import * as db from "~/shared/db";
+import type { Project } from "@webstudio-is/prisma-client";
```

Comment:
this will be a problem, because we rely on the augmented type, which is db.projects.Project

Reply:
I don't understand this

Reply:
you are being fooled by the generic, use the type that you return from loadById

Reply:
also use it here https://github.com/webstudio-is/webstudio-designer/blob/3400b2d33e7df374ce57ac1357023d41e5ff0fd2/apps/designer/app/routes/designer/%24id.tsx#L30